### ISSUE TITLE
Instance-level blocklist sharing

### ISSUE THREAD

**Issue by strugee** [swicg/general#8]
I was just in [a talk][] from the admins of toot.cat discussing what it's been like to monitor that instance, and one thing they mentioned was sharing of instance-level blocklists sharing - that is, it would be super useful to say "I totally trust the admins of foobar.xyz; if they block/silence an instance, I want to do that too." This seems like something that would be nice to standardize as an AP extension.

 []: http://opensourcebridge.org/sessions/2082

**Comment by sandhawke** [comment]
Annnnnd there you have it.   https://w3c.social/@sandro/36170 ends:

> This might end in a long-term split between tightly-policed instances (green zone) and anything-goes instances (red zone). But who controls the policing? It risks being collective punishment.

I'm not sure there's a better solution, but I'm also not really comfortable with promoting a netsplit.

**Comment by nightpool** [comment]
I agree that this is the way things are currently done in the masto community, but also wouldn't be comfortable standardizing it.
